mypy/fastparse.py

```diff
@@ -1205,6 +1283,35 @@ def visit_NameConstant(self, n: NameConstant) -> Type:
         else:
             return UnboundType(str(n.value), line=self.line)
 
+    # Only for 3.8 and newer
+    def visit_Constant(self, n: Constant) -> Type:
+        val = n.value
+        if val is None:
+            # None is a type.
+            return UnboundType(str(val), line=self.line)
```

Comment:
Maybe also `'None'` here? Or we can add a top-level constant `NONE_NAME: Final = str(None)`.